core/src/main/java/com/adobe/aio/workspace/Workspace.java

```diff
@@ -61,19 +66,23 @@ public class Workspace {
   private final String consumerOrgId;
   private final String projectId;
   private final String workspaceId;
+  private final String runtimeNamespace;
+  private final List<String> runtimeActions;
   private final Context authContext;
 
   private Workspace(final String imsUrl, final String imsOrgId, final String apiKey,
                     final String consumerOrgId, final String projectId, final String workspaceId,
-                    Context authContext) {
+                    final String runtimeNamespace, final List<String> runtimeActions, Context authContext) {
     this.imsUrl = StringUtils.isEmpty(imsUrl) ? Constants.IMS_URL : imsUrl;
     this.imsOrgId = imsOrgId;
 
     this.apiKey = apiKey;
     this.consumerOrgId = consumerOrgId;
     this.projectId = projectId;
     this.workspaceId = workspaceId;
-
+    this.runtimeNamespace = runtimeNamespace;
```

Comment:
are these really related to the Workspace ? 
or should we move these config elsewhere  ?